### ISSUE TITLE
How to change url and ssl certs?

### ISSUE THREAD

**Issue by spikyjt** [openid-certification/oidctest#146]
I'd like to change the ssl cert and domain that the test suites run on. As the docs are sparse to say the least, I'd welcome some help. I've tried replacing the certs in `docker/op_test` with my own and modifying both `docker/op_test/config.py` and `docker/op_test/tt_config.py`, but none of this has any effect.

**Comment by zandbelt** [comment]
I think you're looking at a Docker/environment issue then since that is what it should take. What does the browser show about the cert once replaced?

**Comment by spikyjt** [comment]
Yes, I'm running it in Docker as per the very minimal install instructions. The cert I copied in doesn't get used by the docker container. The browser still shows the old self-signed cert and the app is still bound to the default url (https://op-test:60000)

**Comment by zandbelt** [comment]
you'll have to modify at least: https://github.com/openid-certification/oidctest/blob/master/docker/op_test/run.sh

**Comment by spikyjt** [comment]
Yes I can see that is where the port is set, but I don't need to change that, just the host and ssl certs. Seems odd that the two config files in there don't seem to actually be used.

I can see that it isn't really set up for customisation, which is disappointing at best, but it must be possible, as the public site runs on a different domain from the default one. Surely it's expected that people will want to run the suite on their own servers rather than trust the public deployment? Clearly having self-signed certs for that situation is unsatisfactory.
